tests/model_evaluation/test_table_1.py

```diff
@@ -1,22 +1,23 @@
 """Test that table is generated correctly."""
 import pandas as pd
 
-from psycop_model_training.model_eval.base_artifacts.tables.tables import (
-    generate_table_1,
-)
+from psycop_model_training.model_eval.base_artifacts.tables.table_1 import Table1
 from psycop_model_training.model_eval.dataclasses import ArtifactContainer, EvalDataset
 
 
 def test_generate_table_1(synth_eval_dataset: EvalDataset):
     """Test that table is generated correctly."""
+
+    table1 = Table1(
+        eval_dataset=synth_eval_dataset,
+    )
+
     table_spec = ArtifactContainer(
         label="table_1",
-        artifact=generate_table_1(
-            eval_dataset=synth_eval_dataset,
-            output_format="df",
-        ),
+        artifact=table1.generate_table_1(),
     )
 
     output_table = table_spec.artifact
 
-    assert isinstance(output_table, pd.DataFrame)
+    assert not output_table.isnull().values.any()
+    assert output_table.shape == (11, 5)
```

Comment:
```suggestion
    assert output_table.shape == (11, 5)
```

This will change if we add more rows to the table. Instead, I'd check that each of the relevant categories exist in the output_table["categories"] series :-)